charts/openmetadata/values.yaml

```
@@ -116,6 +116,38 @@ global:
           verifyHostname: true
         trustAllConfig:
           examineValidityDates: true
+    samlConfiguration:
+      debugMode: false
+      idp:
+        entityId: ""
+        ssoLoginUrl: ""
+        idpX509Certificate:
+          secretRef: ""
+          secretKey: ""
+        authorityUrl: "http://localhost:8585/api/v1/saml/login"
+        nameId: "urn:oasis:names:tc:SAML:2.0:nameid-format:emailAddress"
+      sp:
+        entityId: "http://localhost:8585/api/v1/saml/metadata"
+        acs: "http://localhost:8585/api/v1/saml/acs"
+        spX509Certificate: 
+          secretRef: ""
+          secretKey: ""
+        callback: "http://localhost:8585/saml/callback"
```

```suggestion
        callback: "http://openmetadata.default.svc.cluster.local:8585/saml/callback"
```

Update to kubernetes service fqn urls